cron/daily-message.js

```diff
@@ -25,31 +25,33 @@ module.exports = {
 
       waitingFor = _.compact(waitingFor);
 
-      // more than 5 days means that it's not release week
-      if (dayDiff === -11) {
-        channel.send(`Release week starts next week on ${moment(dateString).format('YYYY-MM-DD')} are we all prepared? :lts:`);
-      } else if (dayDiff === -4) {
-        channel.send(`:tada: It's release week!! :tada: To see who is done and who isn't you can say '!release done' and I'll tell you who still needs to do something!
+      channels.forEach((channel) => {
+        // more than 5 days means that it's not release week
+        if (dayDiff === -11) {
```

Comment:
I feel like, for Hacktoberfest, there may be a refactoring opportunity with the nested if statement. (e.g. extract function, make early exits)

Reply:
yea seems reasonable 👍 my only objection is that this is a "private bot" i.e. it's not in any public rooms, so encouraging people to contribute to it means they wouldn't see the fruits of their labour in production 😢 

I have no objection to you creating an issue if you felt strongly about it though 👍

Reply:
Thanks, I'll go ahead and create one (I'm anxious that we may not have enough issues for people to work on for the next 30 days).

Even though one may not get to see their work live, I think working on this repo could give people the chance to learn more about how to build a bot for Discord. Maybe this repo can help them.